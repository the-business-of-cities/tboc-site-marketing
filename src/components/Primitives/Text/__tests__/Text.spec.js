import { Line, LineCell, Paragraph} from "../index";
import React from "react";
import { shallow } from 'enzyme';

describe("Line", () => {
  it("renders correctly", () => {
    expect(shallow(<Line />)).toMatchSnapshot();
  });
});

describe("LineCell", () => {
  it("renders correctly", () => {
    expect(shallow(<LineCell />)).toMatchSnapshot();
  });
});

describe("Paragraph", () => {
  it("renders empty correctly", () => {
    expect(shallow(<Paragraph />)).toMatchSnapshot();
  });

  it("renders a sentance correctly", () => {
    expect(shallow(<Paragraph>Something something something</Paragraph>)).toMatchSnapshot();
  });

  it("renders paragraphs correctly", () => {
    expect(shallow(<Paragraph>
      Something something something.

      Something something something.

      Something something something.

      Something something something.
      </Paragraph>)).toMatchSnapshot();
  });
});
