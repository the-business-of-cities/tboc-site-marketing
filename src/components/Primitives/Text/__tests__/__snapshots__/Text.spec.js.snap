// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Line renders correctly 1`] = `
.c0 {
  height: 1.5px;
  width: 100%;
}

<div
  className="c0"
/>
`;

exports[`LineCell renders correctly 1`] = `
<styled.div>
  <styled.div />
</styled.div>
`;

exports[`Paragraph renders a sentance correctly 1`] = `
<div>
  <p
    key="Somet/0"
  >
    Something something something
  </p>
</div>
`;

exports[`Paragraph renders empty correctly 1`] = `<div />`;

exports[`Paragraph renders paragraphs correctly 1`] = `
<div>
  <p
    key="Somet/0"
  >
    Something something something. Something something something. Something something something. Something something something.
  </p>
</div>
`;
